index.html

```diff
@@ -1238,6 +1238,369 @@ <h2>Evidence</h2>
         </p>
 
       </section>
+<section>
+
+      <h1>Subject Not Equal To Holder</h1>
+
+      <p>
+This section describes how the data model supports use cases in which the holder
+ of the credential is not the subject of the credential.
+      </p>
+
+      <p>
+The following diagram depicts those instances when the subject is not equal to 
+the holder.
+     </p>
+
+      <figure>
+        <img style="margin: auto; display: block;" width="75%" src="diagrams/subject-ne-holder.svg">
+        <figcaption style="text-align: center;">
+Subject Holder Relationships in Verifiable Credentials.
+        </figcaption>
+      </figure>
+
+     <p>
+The following sections describe how each of these instances are handled in 
+the data model
+     </p>
+
+
+    <section>
+        <h2>Credential Uniquely Identifies Subject</h2>
+
+     <p>
+In this case, the claim may contain multiple properties that each provide 
+an aspect of the identity of the subject, and which together unambiguously identify
+the subject. In this case, it does not matter who the holder is, since the verifier
+can unambiguously identify the subject of the verifiable credential.
+   </p>
+
+        <pre class="example nohighlight" title="An example of a credential 
+that uniquely identifies a subject">
+
+{
+	"id": "http://dmv.example.gov/credentials/3732",
+	"type": ["Credential", "ProofOfAgeCredential"],
+	"issuer": "https://dmv.example.gov/issuers/14",
+	"issued": "2010-01-01",
+	"claim": {
+		"id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+		"name": "Mr John Doe",
+		"address": "10 Some Street, Anytown, ThisLocal, Country X",
+		"dateOfBirth": "2001-01-01",
+		"gender": "male",
+		"placeOfBirth": "Anytown, ThisLocal, Country X",
+		"nationality": "X",
+	},
+	"proof": { ...
+	}
+}
+        </pre>
+
+
+   </section>
+
+    <section>
+       <h2>Subject Delegates to Holder</h2>
+
+     <p>
+This is supported in the data model in two different ways: through the use of either recursion or verifiable presentations. Both methods involve the subject issuing a new verifiable credential to the delegated holder.
```

Comment:
Why doesn't the subject just issue an ocap to the holder for a very specific grant of authority. Delegating a credential is dangerous because you're not also delegating what that delegation can be used for. For example, if I delegate a driver's license to you... does that mean you can use it to open a bank account online? or open a line of credit? Or get into a bar? How are the limitations for that grant put into place? This is why we have OCAPs.

Reply:
People do not delegate driving licenses, or age credentials. These will clearly be marked as not delegatable (in fact this should be the default. The issuer should clearly mark a VC as delegatable if it wants it to be)

Reply:
I'd argue that people don't delegate roles either. They grant permission to operate on their behalf, given their authority as a particular role in an organization. I don't say "here, have my key card so you can do anything I can" (ambient authority)... I say "here, have my key card so you can do X, and only X". That's an OCAP.

Reply:
I guess we just have to disagree over this point. Role delegation is very common in RBAC systems, and VCs are a new way of handing out roles to employees. (Our existing software, PERMIS (www.openpermis.org) uses X.509 ACs to hold roles. It would be nice to move to VCs).